jwql/tests/test_data_containers.py

```diff
@@ -49,18 +134,197 @@ def test_get_all_proposals():
     assert len(proposals) > 0
 
 
-@pytest.mark.skipif(ON_GITHUB_ACTIONS, reason='Requires access to central storage.')
-def test_get_expstart():
+@pytest.mark.parametrize('untracked,input_suffixes,expected',
```

Comment:
Good catch on these tests that we don't need to skip on github actions. Your updated tests match the list I created the other day. I'll attach the relevant issue to this PR so that it will be closed.

Reply:
Oh good, thanks.  I didn't realize there was an issue for this.